static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/launch.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd /mnt/SDCARD/App/Screenshots_Viewer
+/mnt/SDCARD/.tmp_update/bin/infoPanel -d /mnt/SDCARD/Screenshots --show-theme-controls
```

Comment:
Showing the theme controls on top of screenshots is not desired. You either need to resize the screenshots or implement a toggle on Y (the same as GS).

Reply:
Fixed by [645d6d6](https://github.com/OnionUI/Onion/pull/115/commits/645d6d6d5d28f73b0b5dfd75b30cebd9b6c9048c)